ds-ux-guidelines/src/ds-components/layout/layout.scss

```diff
@@ -1,8 +1,9 @@
+@import "~cbp-ds/cbp-ds.min.css";
```

Comment:
getting the following error when I try to build: 

Generating JavaScript bundles failed

Can't resolve 'cbp-ds/cbp-ds.min.css' in '/Users/aeijspn/Desktop/cbp-theme/ds-ux-guidelines/src/ds-components/layout'

File: src/ds-components/layout/layout.scss

Reply:
ensure you get the node package cbp-ds, npm install 
should have mentioned in the instructions, you can confirm it is there in the node_modules folder

Reply:
thanks for the clarification, this worked for me